### PR TITLE
Added PHP 8 Support

### LINKED ISSUE #35: PHP8
When are you planning to add php 8 support?

Regards,
Gregor

### PR DESCRIPTION
**Pull request by fulopattila122**
Enabled PHP 8.0+ in composer, added relevant adjustments to CI config.

Fixes #35

### CODE REVIEW

File: .php_cs.cache [hunk 1]
```diff
@@ -1 +1 @@
...
...rts":true,"not_operator_with_successor_space":true,"trailing_comma_in_multiline_array":true,"phpdoc_scalar":true,"unary_operator_spaces":true,"binary_operator_spaces":true,"blank_line_before_statement":{"statements":["break","continue","declare","return","throw","try"]},"phpdoc_single_line_var_spacing":true,"phpdoc_var_without_name":true},"hashes":{"src\/MailTemplatesServiceProvider.php":2039851809,"src\/Exceptions\/CannotRenderTemplateMailable.php":2298534153,"src\/Exceptions\/MissingMailTemplate.php":1735637041,"src\/Interfaces\/MailTemplateInterface.php":1572255043,"src\/TemplateMailable.php":1639270651,"src\/TemplateMailableRenderer.php":3638620062,"src\/Models\/MailTemplate.php":3521765837,"tests\/CustomMailTemplateTest.php":1948617665,"tests\/MailTemplateModelTest.php":997461801,"tests\/stubs\/Mails\/BadLayoutMail.php":1561344492,"tests\/stubs\/Mails\/BasicMail.php":1882642824,"tests\/stubs\/Mails\/CustomTemplateModelMail.php":588745898,"tests\/stubs\/Mails\/LayoutMail.php":4198379676,"tests\/stubs\/Models\/CustomMailTemplate.php":679533390,"tests\/stubs\/Models\/LayoutMailTemplate.php":2446277769,"tests\/TestCase.php":4251254877,"tests\/TemplateMailableTest.php":2145745534}}
```
Comment (freekmurze) [inline .php_cs.cache:1]: Please remove this from the PR

**Review by freekmurze — CHANGES_REQUESTED** [review]
Thanks for your work on this.

File: .php_cs.cache (hunk 1 above)
Comment (fulopattila122) [reply .php_cs.cache]: Done ✔

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**